package com.maths;

public class Exception_Task {
	public static void main(String[] args) {
		
	
	int a=3/0;
	System.out.println(a);

	}
}

